canonical-websites/www.ubuntu.com
# 1615 cloud plans and pricing

## Done

* update /cloud/plans-and-pricing to vbt
* hero is accent colour, like on live
* landscape table layout required a change… please review
* collapsed some vertical space

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [page](http://0.0.0.0:8001/cloud/plans-and-pricing)
- compare to the [design](https://www.dropbox.com/work/00_web_team/_ubuntu.com/new%20projects/vanilla%20update/Patterns%20inventory/2.%20Cloud?preview=cloud-plans-and-pricing.png)

## Issue / Card

Fixes #1615
